tests/model_serving/model_runtime/vllm_omni/conftest.py

```
@@ -0,0 +1,281 @@
+    """
+    runtime_image = pytestconfig.option.vllm_omni_runtime_image
+    if not runtime_image:
+        return None
+    return runtime_image
+
+
+@pytest.fixture(scope="session")
+def skip_if_no_vllm_omni_multi_gpu(supported_accelerator_type: str | None) -> None:
+    """Skip test unless the cluster provides at least 2 NVIDIA GPUs."""
+    if not supported_accelerator_type or supported_accelerator_type.lower() != AcceleratorType.NVIDIA:
+        pytest.skip(f"vLLM-Omni requires 2× NVIDIA GPUs. Found accelerator: '{supported_accelerator_type or 'None'}'.")
```

Ruff (0.16.1)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)